plugins/feeds/public/abusech_malwarebazaar.py

```diff
@@ -63,14 +63,22 @@ def analyze(self, block):  # pylint: disable=arguments-differ
         context["imphash"] = block["hash"]["imphash"]
         context["ssdeep"] = block["hash"]["ssdeep"]
 
-        if block.get("hash", []):
+        if block.get("sha256_hash", ""):
```

Comment:
No need to specify a second argument here:

```
$ python3
Python 3.7.7 (default, Mar 10 2020, 15:43:33)
[Clang 11.0.0 (clang-1100.0.33.17)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> {}.get('asd') is None
>>> True
```

Reply:
Also, it's even better to use a guard statement:

```python
if not block.get('sha256_hash'):
  return
malware_file = File.get_or_create(...)
```

Reply:
updated